src/dune_rules/install_rules.ml

```diff
@@ -193,6 +192,8 @@ end = struct
           in
           make_entry ?sub_dir Lib path)
     in
+    let* melange_runtime_entries = additional_deps lib.melange_runtime_deps in
+    let* public_headers = additional_deps lib.public_headers in
```

Comment:
perhaps this can be done in parallel? `let* ... and* ... in`

Reply:
Done. There's many opportunities to do such optimizations in the install rules.

Reply:
Is this really an optimisation? I thought and* is just implemented as a bind.

Reply:
Look at the definition of `and*` in fiber.